Make mention of the new CiliumNetworkPolicy in the appropriate documentation (#197)
appuio/appuio-cloud-docs: docs/modules/ROOT/pages/how-to/remove-default-networkpolicies.adoc

= Remove Default NetworkPolicies

In every namespace 2 `NetworkPolicies` and one `CiliumNetworkPolicy` are created and maintained by {product}:

> . `networkpolicy/allow-from-other-namespaces`: This policy allows the Router and other system components to connect to the pods.

Reviewer: ```suggestion
. `networkpolicy/allow-from-other-namespaces`: This policy allows system components such as the monitoring stack to connect to the pods.
```

. `networkpolicy/allow-from-same-namespace`: This policy allows connections between pods in the same namespace.

> . `ciliumnetworkpolicy/allow-from-cluster-nodes`: This policy allows cluster nodes to connect to the pods.

Reviewer: ```suggestion
. `ciliumnetworkpolicy/allow-from-cluster-nodes`: This policy allows the OpenShift ingress router and other system components that run in the host network namespace to connect to the pods.
```
